### DISCUSSION TITLE
[Feature Request] FIDO2, 2FA

### DISCUSSION THREAD

**Discussion by DarkArc** [standardnotes/forum#2616] [Suggestions]
It would be great if more than TOTP was supported for 2FA, particularly, I'd like to see support for FIDO2 2FA keys.

**Comment by JaspalSuri** [comment]
Thanks for the feedback, I'll include your +1. :)

**Comment by joshcsullivan** [comment]
+1 for this feature.

I've recently switched a number of 2FA to FIDO2/WebAuthn, Standard Notes is the one of two that I'm stuck using Yubico OTP.

**Comment by DoubleMCA** [comment]
+1
